packages/tools/pluggable-widgets-tools/configs/webpack.config.common.js

```diff
@@ -78,7 +78,7 @@ const previewConfig = {
                 loader: "ts-loader",
                 options: {
                     compilerOptions: {
-                        module: "CommonJS"
+                        target: "es6"
```

Comment:
Are you doing it because of drop of IE? If so, then it is better to change this config in `tsconfig.base`

Reply:
Nope, because as the module is esnext now the output apparently is not supported in editorConfig and editorPreview (SP side). For some reason CommonJS as a module was breaking, we can either leave empty without options or define the target for those files. Tested in 8.15.2 and master

Reply:
This does not really add up. Module system is transpiled by webpack, that is why we can safely change it. Target is not, so making this change is actually very dangerous.

Reply:
Discussed and changed